### PR TITLE
Web3 | AST - purge

### PR DESCRIPTION
**Pull request by ap-justin**
Ticket(s):
-

## Explanation of the solution
* removed all web3 wiring except related token transfer and balance
* remove all wiring/UI related to AST
* test donation still works ✅

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/pages/Admin/Charity/EditProfile/index.tsx [hunk 1]
```diff
@@ -42,12 +41,11 @@ export default function EditProfile() {
...
 
-function FormWithContext(props: TProfile) {
-  const { txResource, id, owner } = useAdminContext<"charity">(ops);
-
+function FormWithContext(props: TProfile & { id: number }) {
+  //
   const init: EndowmentProfileUpdate = toProfileUpdate({
     type: "initial",
-    data: { ...props, id, owner },
+    data: { ...props, id: props.id, owner: "is not relevant anymore" },
```
Comment (ap-justin) [inline src/pages/Admin/Charity/EditProfile/index.tsx:48]: @SovereignAndrey, @jp-mariano , I assume there would be a refactor on edit profile endpoint once web2 authentication is in place

File: src/pages/Gift/Purchase/Submit/index.tsx [hunk 2]
```diff
@@ -10,19 +10,10 @@ import { SubmitStep, setStep } from "slices/gift";
 import { humanize } from "helpers";
 import { appRoutes } from "constants/routes";
 import CompleteBtn from "./CompleteBtn";
-import { estimateTx } from "./estimateTx";
 
 export default function Submit(props: WithWallet<SubmitStep>) {
   const dispatch = useSetter();
-  const [estimate, setEstimate] = useState<EstimateStatus>("loading");
```
Comment (ap-justin) [inline src/pages/Gift/Purchase/Submit/index.tsx:17]: not use if giftcards still has web3 nature so just left most of the existing UI and web3 wiring

File: src/services/types.ts [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
-  BridgeFees,
+  ApplicationStatus,
   EndowmentProfile,
   EndowmentProfileUpdate,
   WalletProfile,
 } from "types/aws";
-import { EndowmentDetails } from "types/contracts";
-import { AccountType, EndowmentType } from "types/lists";
-import { Endowment } from "types/subgraph";
+import { EndowmentType } from "types/lists";
```
Comment (ap-justin) [inline src/services/types.ts:8]: still used in marketplace query params, might be removed on later endpoint versions

File: src/types/tx.ts [hunk 4]
```diff
@@ -148,100 +126,19 @@ type Tx<T extends Tupleable, M> = {
...
-  "multisig.revoke-tx": Tx<ID, never>; //no meta
-  "multisig.execute-tx": Tx<ID, never>; //no meta
-  "multisig.change-threshold": Tx<{ threshold: number }, ThresholdMeta>;
-  "multisig.change-auto-execute": Tx<{ autoExecute: boolean }, never>; //no need for meta
-  "multisig.change-duration": Tx<{ duration: number }, DurationMeta>; //no need for meta
-
-  "multisig/review.confirm-prop": Tx<ID, never>; //no meta
-  "multisig/review.execute-prop": Tx<ID, never>; //no meta
-
   "erc20.transfer": Tx<Transfer, TransferMeta>;
```
Comment (ap-justin) [inline src/types/tx.ts:132]: still used in `donations`

File: src/pages/Gift/Purchase/Submit/index.tsx (hunk 2 above)
Comment (SovereignAndrey) [reply src/pages/Gift/Purchase/Submit/index.tsx:17]: @ap-justin Gift Cards will be 100% AWS driven now. @jp-mariano has removed the `deposit_id` field already in AWS that the contracts used. Now, a giftcard balance will simply need to be reduced as the user spends it. This'll need some re-thinking of the UI and some changes in AWS to make sure we are correctly tracking remaining spendable balances (ie. $100 USD initially finalized and later claimed by a user with the code could be spend in multiple chunks like two donations "with a gift card" of $25 & $75 each). 

I'd suspect we end up adding GiftCards as an Payment Option on donations, under an "Other" category, coming after existing "Credit Card" and "Crypto" tabs. An "Other" tab is where we'll add "DAF" and "Stocks" for those donations options (eventually).

File: src/services/types.ts (hunk 3 above)
Comment (SovereignAndrey) [reply src/services/types.ts]: There's still a need to distinguish types I think, like DAFs, once we're ready for them.

File: src/pages/Admin/Charity/EditProfile/index.tsx (hunk 1 above)
Comment (SovereignAndrey) [reply src/pages/Admin/Charity/EditProfile/index.tsx:51]: @ap-justin Yeah, `owner` will fall away, to be replaced by auth checks if a given `User` is authorised/linked to an `Endowment`.

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**